src/Parallel/NodeLock.hpp

```diff
@@ -3,71 +3,43 @@
 
 #pragma once
 
-#include <converse.h>
```

Comment:
I think you need this include for `CmiLock`

Reply:
Ah, oops, I think you're correct.